We (OE) install slang into /usr/include so we never need to look into 
/usr/include/slang/. We never want to look into a hardcoded path like this
since it triggers host infection issues. For now, simply remove this
since it causes us problems.

Upstream-Status: Pending (would need rework)

Index: tools/perf/Makefile
===================================================================
--- linux.orig/tools/perf/Makefile	2012-08-07 10:29:43.020149620 +0000
+++ linux/tools/perf/Makefile	2012-08-07 10:30:08.128148098 +0000
@@ -504,7 +504,7 @@
 		BASIC_CFLAGS += -DNO_NEWT_SUPPORT
 	else
 		# Fedora has /usr/include/slang/slang.h, but ubuntu /usr/include/slang.h
-		BASIC_CFLAGS += -I/usr/include/slang
+		# BASIC_CFLAGS += -I/usr/include/slang
 		EXTLIBS += -lnewt -lslang
 		LIB_OBJS += $(OUTPUT)util/ui/setup.o
 		LIB_OBJS += $(OUTPUT)util/ui/browser.o
